frontend/src/modules/machinery/components/CaighdeanTranslation.js

```diff
@@ -16,12 +16,12 @@ export default function CaighdeanTranslation() {
                 <a
                     className='translation-source'
                     href='https://github.com/kscanne/caighdean'
-                    title='Visit Caighdean Machine Translation'
+                    title='VISIT CAIGHDEAN MACHINE TRANSLATION'
```

Comment:
You won't need to transform the `title` here as in this case it is an attribute and not a prop. This is true for the other instances of `translation-source` as well.